parm/soca/soca_fix_stage.yaml.j2

```diff
@@ -1,3 +1,5 @@
+mkdir:
+- "{{ DATA }}/INPUT"
 ######################################
```

Comment:
Can you add a `TODO` here specifying that this is resolution dependent ... I need to address this on my side too.